presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -482,34 +487,40 @@ Deprecated Property Name                  New Property Name
 ``metrics_max_inferred_column``           ``write.metadata.metrics.max-inferred-column-defaults``
 =======================================   ===============================================================
 
+Presto C++ Support
+^^^^^^^^^^^^^^^^^^
+
+Table properties are not supported in Presto C++ because write operations have not been implemented.
```

Comment:
I think we should consider re-wording this. It's not quite clear what the statement "Table properties are not supported ..." means. Does that mean reading them? writing them? considering them during writes? I think we should be a bit more specific

Reply:
> I think we should consider re-wording this. It's not quite clear what the statement "Table properties are not supported ..." means. Does that mean reading them? writing them? considering them during writes? I think we should be a bit more specific

I think what @ZacBlanco said makes sense. @agrawalreetika Will you be able to try them and update the doc separating the read and write